### DISCUSSION TITLE
Testimonial- New Feature

### DISCUSSION THREAD

**Discussion by smsalmanshahAN** [an-nadaaa/nda-front#108] [Feedback]
If we can have tags applied to testimonials then it will help in populating the testimonial in the appropriate location automatically

So for example we have 20 test  testimonials 2 are for wells

If they are tagged they can be automatically added to the wells project page

So later we add a new testimonial and whike adding the testimonial the admin adds the project tag and it will automatically be added to that project page and it is not required to manually select the testimonials to add on each project

**Comment by an-nadaa** [comment]
I like this idea. i haven't thought of adding testimonials to the individual project pages. we can definitely do that
